README.md

```diff
@@ -30,21 +30,49 @@ That's it.
 
 ### How it works
 
-Basically this plugin executes the following steps:
+Basically, this plugin executes the following steps:
 
 - This plugin search for `phpcodesniffer-standard` packages in all of your currently installed Composer packages.
 - Matching packages and the project itself are scanned for PHP_CodeSniffer rulesets.
 - The plugin will call PHP_CodeSniffer and configure the `installed_paths` option.
 
-### Calling the plugin directly 
+### Example project
 
-In some circumstances it is desirable to call this plugin's functionality 
+The following is an example Composer project and has included
+multiple `phpcodesniffer-standard` packages.
+
+```json
+{
+    "name": "dealerdirect/example-project",
+    "description": "Just an example project",
+    "type": "project",
+    "require": {},
+    "require-dev": {
+        "dealerdirect/phpcodesniffer-composer-installer": "*",
+        "wimg/php-compatibility": "*",
+        "wp-coding-standards/wpcs": "*",
+        "object-calisthenics/phpcs-calisthenics-rules": "*"
```

Comment:
I would suggest placing the requirements in alphabetical order, specifically because `"config": {"sort-packages": true}` is becoming more and more popular